introduction_to_amazon_algorithms/jumpstart_text_summarization/Amazon_JumpStart_Text_Summarization.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>@julkroll do we need have these files in sagemaker-sample-files to unify our data sources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
These are not static sources and keep getting updated as we update our scripts/models etc.